examples/keymanager/keymanager_secret/main-password.tf

```diff
@@ -0,0 +1,7 @@
+resource "vkcs_keymanager_secret" "password" {
```

Comment:
why do we need this?

Reply:
we do not have cases for this secret type

Reply:
i removed it